module.js

```diff
@@ -0,0 +1,136 @@
+const catchA = function () {
+    let name = '';
+    let start = false;
+    let score= 0 ;
+    let level=1 ;
+    let iphones=[];
+    let idCounter=1
+    let timer = 2
+   
+    const randomColor = function () {
+        let ranNum = function () {
+            return Math.floor(Math.random() * 255)
+        }
+        let color = `rgb(${ranNum()},${ranNum()},${ranNum()})`
+        return color
+    }
+
+
+
+    const addIphone = function (num) { 
+        let ranNum = function () {
```

Comment:
try not to have full functions inside of another full function. take the `ranNum` function outside of the `addIphone` function and just invoke it inside the `addIphone` function.